granulate_utils/logging/__init__.py

```diff
@@ -0,0 +1,138 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+import logging
+import threading
+import time
+import uuid
+from datetime import datetime
+from logging import Handler, LogRecord
+from typing import Any, Dict, List, Tuple, Union
+
+import backoff
+import requests
+from requests import RequestException
+
+REQUEST_MAX_TRIES = 5
+
+logger = logging.getLogger(__name__)
+
+
+class BatchRequestsHandler(Handler):
```

Comment:
How is this used? Would add documentation on how to use it and maybe even create a function to help this process **if** its complicated

Reply:
Most basic usage would be something like this:
```python
handler = BatchRequestsHandler("localhost:61234")
logging.basicConfig(handlers=[handler])
```

More fine-grained would probably look like this:
```python
logger = logging.getLogger("gprofiler")
handler = BatchRequestsHandler("localhost:61234")
handler.setLevel(logging.DEBUG)
logger.addHandler(handler)
```

Reply:
Added docs.